samples/authenticator-app/.gitignore

```diff
@@ -0,0 +1,42 @@
+# Learn more https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files
```

Comment:
Change the wording in camera permission, and add configurable icon for modal wityh camera icon.

Reply:
addressed with https://github.com/asgardeo/javascript/pull/195/commits/67fb2dcb95a400eae70ea951547cda82e6de6ace